ethereum/PFP.sol

```diff
@@ -64,19 +64,27 @@ library MerkleProof {
 contract PFP is ERC721A, Ownable, VRFConsumerBase {
     using Strings for uint256;
     
-    uint256 public MAX_TOKENS = 20000;
-    string public baseExtension = ".json";
-    string private _baseURIextended;
+    uint256 public constant MAX_TOKENS = 20000;
+    string public constant BASE_EXTENSION = ".json";
+    string public _baseURIextended;
```

Comment:
Made this public to support the locking of the token URI after reveal (See below)

With this public you can review the value and really really make sure it's all going to work before you reveal and lock it forever in place.